functions/lessonNotifications.js

```diff
@@ -94,6 +94,113 @@ module.exports = {
     }
   },
 
+  notifyStudentInstantLessonConfirmed: async (lesson) => {
+        const student = String(lesson.studentUid).split(",")[0];
+
+        try {
+          // Fetch the tutor's profile
+          const studentQuery = await db.collection("profiles").where("uid", "==", studentUid).limit(1).get();
+
```

Comment:
not so trivial to get the profile 🤣, I think that the comment should be fetch the student profile and not the tutor.